python/ml4ir/base/data/tfrecord_reader.py

```diff
@@ -345,7 +345,10 @@ def read(
     # Parse the protobuf data to create a TFRecordDataset
     dataset = data.TFRecordDataset(tfrecord_files)
     if parse_tfrecord:
-        dataset = dataset.map(parse_fn).apply(data.experimental.ignore_errors())
+        # Parallel calls set to AUTOTUNE: improved training performance by 40% with a classification model
```

Comment:
Added this additional improvement. So now with both improvement, training time is improved by 60%

Reply:
this is amazing!